vite.config.js

```diff
@@ -3,6 +3,9 @@ import tailwindcss from '@tailwindcss/vite';
 import laravel from 'laravel-vite-plugin'
 import { wordpressPlugin, wordpressThemeJson } from '@roots/vite-plugin';
 
+// Set APP_URL for Laravel Vite plugin
+process.env.APP_URL = 'http://example.test';
```

Comment:
I would check whether the value exists before setting it here.
```suggestion
// Set APP_URL if it doesn't exist for Laravel Vite plugin
if (! process.env.APP_URL) {
  process.env.APP_URL = 'http://example.test';
}
```